distributed/l2_distance/l2_distance_heat.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import datetime
+import heat as np
+
+def l2_distance(X1, X2):
+        sub = X1-X2
+        sq = np.square(sub)
+        sum = np.sum(sq)
+        d = np.sqrt(sum)
+        return d
+
+def initialize(size, dims):
+        np.random.seed(7777777)
+        return (np.random.random((size, dims)), np.random.random((size, dims)))
```

Comment:
heat requires a split-axis when generating new arrays, otherwise data/compute is just replicated.